src/sfctl/tests/cluster_test.py

```diff
@@ -43,3 +48,25 @@ def test_select_cert_and_aad(self):
         with self.assertRaises(CLIError):
             sf_c.select_arg_verify('http://test.com:190800', 'test.crt',
                                    'test.key', None, None, True, False)
+
+class ClusterScenarioTests(ScenarioTest):
+    """Cluster scenario tests"""
+
+    def __init__(self, method_name):
+        cli_env = cli()
+        super(ClusterScenarioTests, self).__init__(cli_env, method_name)
+
+    @patch('sfctl.config.CLIConfig', new=MOCK_CONFIG)
+    def test_show_cluster(self):
+        """Ensure that the correct message is returned when a cluster is set or not set"""
+        old_endpoint = environ.get('SF_TEST_ENDPOINT')
+
+        environ['SF_TEST_ENDPOINT'] = ''
```

Comment:
Other minor nit, if the implementation of set endpoint ever changes this will fail so you may want to mock the object that calls set and then call it yourself.

Reply:
I'll put this one on the backlog :)